compose/runtime/runtime/src/uikitMain/kotlin/androidx/compose/runtime/Trace.uikit.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.runtime
+
+import androidx.compose.ui.uikit.utils.CMPOSLogger
+import androidx.compose.ui.uikit.utils.CMPOSLoggerInterval
+
+
+/**
+ * Enables iOS OS logging for the `androidx.compose.runtime` APIs.
+ *
+ * Tracing allows logging detailed information about the Compose UI framework, which can be further
+ * analyzed using the XCode Instruments tool.
+ *
+ * This method is an [ExperimentalComposeApi], which means it is subject to change without notice in major, minor, or patch releases.
+ *
+ * @see ExperimentalComposeApi
+ */
+@ExperimentalComposeApi
+fun enableTraceOSLog() {
+    if (traceImpl == null) {
+        traceImpl = CMPOSLogger(categoryName = "androidx.compose.runtime")
```

Comment:
It seems it should work on all Apple platforms - macOS, tvOS, watchOS

Reply:
No, this class is implemented in UIKitUtils framework, which is built for iOS now.